independent-publisher-connectors/MaintenanceRequestOxmaint/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#f7f7f7",

```

Comment:
Kindly refer below link and change the iconBrandColor, which is standard color for Independent Publisher.